hera_cal/hera_cm.py

```diff
@@ -0,0 +1,198 @@
+import numpy as np
+import pandas as pd
+import os
+import aipy
+import pyuvdata.utils as uv_utils
+from astropy.time import Time
+from dateutil.parser import parse
+
+cm_p_m = 100
+#locations_file = os.path.join(os.path.dirname(__file__), 'data/hera_ant_locs_05_16_2017.csv')
+
+# this sets when to find the antenna positions for (they changed as HERA was built)
+# this should not be later than the date that the file was pulled from M&C
+array_epoch_jd = 2457458
```

Comment:
A default, hard coded date is included in the cal file to be somewhat consistent with  historical use of cal files. The way we usually used cal files was that a cal file corresponded to a specific date or date range. 

Here we have this as the default so that the cal file can still be used in the normal way.  I've changed the get_aa function call so that all non-standard inputs revert to defaults within the cal file so that the custom get_HERA_aa function is not required to use this cal file.